app/services/driver_service.py

```
@@ -0,0 +1,28 @@
+from typing import List, Tuple, Optional
```

Ruff (0.11.9)

`typing.Tuple` imported but unused

Remove unused import

(F401)